### PR TITLE
Added WYH L12864 LCD from Alfawise Ex8 printer. Resolves #22856

### LINKED ISSUE #22856: [FR] Add support of Alfawise Ex8 LCD. (ST7565_64128N SPI)
### Is your feature request related to a problem? Please describe.

_No response_

### Are you looking for hardware support?

_No response_

### Describe the feature you want

I need to connect LCD from Alfawise Ex8 3d printer marked as WYH L12864 COG (see attachment). This is ST7565_64128N based board. I get this LCD worked as AZSMZ_12864 (#define AZSMZ_12864 in the Configuration.h) but there is picture offset for 4 pixels from right to left. See attachment. Others ST7565 lcds for example MAKRPANEL or VIKI2 also works with same result. Where is a place to correct this problem and add support for this LCD to Marlin?  
IMG_8716

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by VragVideo**
There is Alfawise Ex8 3d printer with custom LCD display marked as WYH L12864 COG. It has unique pinout and picture is shifted by 4 pixels if it is connected without changes. This patch allows to use this LCD with standard boards. Currently implemented Big Tree Tech SKR 1.3 and Big Tree Tech SKR Pro boards.

### Requirements

LCD from Alfawise Ex8 printer marked as WYH L12864 COG. Big Tree Tech SKR 1.3 or Big Tree Tech Pro boards.

### Benefits

Patch allows to use LCD on these boards and allows easily add support to other boards.

### Configurations

I tested configuration on my EX8 with BTT Skr 1.3 with TMC 2208 drivers and Triangle BMG Aero extruder. 
### Related Issues
N/A
<!-- Does this PR fix a bug or fulfill a Feature Request? Link related Issues here. --
[Ex8.zip](https://github.com/MarlinFirmware/Marlin/files/7268767/Ex8.zip)
IMG_8716
>

### CODE REVIEW

File: Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h [hunk 1]
```diff
@@ -252,6 +252,38 @@
     #define LCD_PINS_ENABLE          EXP1_05_PIN
     #define LCD_PINS_D4              EXP1_07_PIN
 
+  #elif ENABLED(WYH_L12864)
+
+  //#error "CAUTION! WYH_L12864 requires wiring modifications. Comment out this line to continue."
```
Comment (thisiskeithb) [inline Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h]: This line shouldn't be commented out by default if you want to alert the user of the required wiring.

Comment (VragVideo) [reply Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h]: Thank you! Fixed

**Comment by thinkyhead** [comment]
It looks thorough to me, with good scary alerts – just in time for Halloween. I'll apply changes to configurations and merge it pretty soon.

**Review by VragVideo — COMMENTED** [review]
Thank you for fixing my bad English

**Pull request merged**